test/acceptance/tests.py

```diff
@@ -987,8 +1015,8 @@ def setUp(self):
 
     @retry()
     @attr('acceptance')
-    @ddt.data(True, False)
-    def test_train_self_peer_staff(self, peer_grades_me):
+    @ddt.data(False)
+    def test_train_self_peer_staff(self, peer_grades_me):  # TODO: can't run with "True" due to TNL-3957
```

Comment:
The "True" case wasn't actually covering what it was supposed to. The bug in verify_submission_has_peer_grade meant that the learner may not have been graded by peers. But due to TNL-3957, there is no way to tell if a learner has been graded by peers in the full grading workflow.